etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -67,8 +68,6 @@ source kayobe-env --environment ci-aio
 
 kayobe control host bootstrap
 
-kayobe playbook run etc/kayobe/ansible/growroot.yml
```

Comment:
Please keep growroot - it does some additional things

Reply:
Okay, the change is now reverted